src/components/ClipboardAwarePasteButton.tsx

```diff
@@ -17,7 +17,9 @@ export default function ClipboardAwarePasteButton({
   const { t } = useTranslation()
 
   useLayoutEffect(() => {
-    LayoutAnimation.easeInEaseOut()
+    if (Platform.OS !== 'android') {
+      LayoutAnimation.easeInEaseOut()
+    }
```

Comment:
yeah i'm not really sure what's going on here, but maybe `setLayoutAnimationEnabledExperimental` means "enable it to work kind of well" instead of "enable it to work at all"? idk but for me turning off the animation makes all the difference